### PR TITLE
Fix warnings for FilesDisplayActivity

### PR DESCRIPTION
**Pull request by alperozturk96**
- [x] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 1]
```diff
@@ -2338,44 +2291,6 @@ public void showFiles(boolean onDeviceOnly) {
         }
     }
 
-    @Subscribe(threadMode = ThreadMode.BACKGROUND)
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java:2341]: @tobiasKaminsky might know this better, but I expect these functions to be used by the event bus, so the code checks are false positives.

**Review by AndyScherzinger — CHANGES_REQUESTED** [review]
see comment, best to check with Tobias

File: app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZetaTom) [reply app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java:2341]: I've done some testing and found out, that these functions are indeed called when an event is dispatched on the eventbus.

For example, `public void onMessageEvent(final SearchEvent event)` is called, when the user switches to the *Favourites* tab from the main *All files* view.

**Review by tobiasKaminsky — APPROVED** [review]

**Comment by alperozturk96** [comment]
Outdated PR, too much diff to merge it.

**Pull request closed without merging**